packages/account-kit/cli/utils/deployLocal.ts

```diff
@@ -0,0 +1,43 @@
+import { Address, Hex, createWalletClient, http, publicActions } from "viem";
+import { privateKeyToAccount } from "viem/accounts";
+import { foundry } from "viem/chains";
+
+// Anvil dev private key #7, automatically funded when running `anvil`
+export const devPrivateKey = "0x4bbbf85ce3377467afe5d46f804f221813b2bb87f24d81f60f1fcdbf7cbf4356";
+const account = privateKeyToAccount(devPrivateKey);
+
+export const deployerClient = createWalletClient({
+  account,
+  chain: foundry,
+  transport: http("http://127.0.0.1:8545"),
+}).extend(publicActions);
+
+export const deterministicDeployer: Address = "0x4e59b44847b379578588920ca78fbf26c0b4956c";
```

Comment:
wonder if we should expose some of our deployment utils from cli package

Reply:
or expand our deploy logic (maybe via mud config?) to allow including external contracts to deploy alongside

Reply:
Agree something in the mud config to deploy extra contracts might be helpful (this is also what `PostDeploy.s.sol`'s original intention was). In this specific case where we're creating a "standalone" script to run local account abstraction infra adding it to the mud deployer would probably not help though. Agree exposing `ensureContract` could help deduping some logic here, but would need some more changes (ie accepting a salt, making `deployedBytecodeSize` optional) so i'd rather add a follow up issue for that